### PR TITLE
update lighthouse docker file to use supported node version

### PR DESCRIPTION
**Pull request by andrewscfc**
Overall changes
======
- Updates Lighthouse `Dockerfile` to use Node 18
- Adds a new multi-stage step to remove the V8 compiled build from the image

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

**Comment by amoore108** [comment]
Added in a [multi-stage](https://docs.docker.com/build/building/multi-stage/) step to prevent the entire V8 compiled build getting bundled in. Without doing this, the image size was approximately 5GB. With this, it comes down to around 890MB: https://stackoverflow.com/questions/61518107/docker-image-with-v8-compiled-reached-almost-5gb-what-am-i-doing-wrong?rq=3

**Review by karinathomasbbc — APPROVED** [review]

File: lighthouse/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,11 @@
-FROM node:12-alpine
+FROM node:18-alpine
 
 RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories \
     && apk add chromium
 
 RUN npm install -g @lhci/cli@0.6.x
```
Comment (eagerterrier) [inline lighthouse/Dockerfile]: do we want to look at updating `@lhci/cli`? That version is 3 years old...  https://www.npmjs.com/package/@lhci/cli/v/0.6.1

Comment (amoore108) [reply lighthouse/Dockerfile]: Defo. I'll take a look now. Good catch 👍

Comment (amoore108) [reply lighthouse/Dockerfile]: Updated the version to `0.12.x` and ran locally. Seems to be fine.

**Review by eagerterrier — APPROVED** [review]
T-Shirt-Image-2_1-1

**Pull request merged**